.wordlist.txt

```diff
@@ -85,6 +87,8 @@ HBM
 heterogenous
 hipBLAS
 hipBLASLt
+hipcc
+HIPCC
```

Comment:
See: https://github.com/RadeonOpenCompute/ROCm/pull/2078#issuecomment-1520459111

HIPCC, in full capitalization is okay, IMHO.

Reply:
Done. Removed hipcc and kept HIPCC